client/src/components/Collections/FixedIdentifierDatasetCollectionElementView.vue

```diff
@@ -0,0 +1,29 @@
+<script setup lang="ts">
+import type { HDASummary } from "@/api";
+import localize from "@/utils/localization";
+
+interface Props {
+    element: HDASummary | undefined;
```

Comment:
```suggestion
    element?: HDASummary;
```
Subjectively prettier and more consistent :)